modules/security_hub/main.tf

```diff
@@ -14,6 +14,14 @@ resource "aws_securityhub_member" "default" {
   email      = each.value
   invite     = true
```

Comment:
Have you tested setting `invite = false`? I believe the invite is useful when adding accounts via email but since we are using Organizations, I believe this should be set to `false`.

Reply:
Ferando and I did a little more research about the invite functionality and this is only needed when inviting member accounts that are not in our organization. We will remove this setting for now (since the default is false). More information: https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-accounts-add-invite.html